### DISCUSSION TITLE
Elastalert Container Missing/Broken After Update to 2.3.120

### DISCUSSION THREAD

**Discussion by Mfisher925** [Security-Onion-Solutions/securityonion#7878] [Unsupported Versions] [answered]
Today I updated our distributed deployment from the Stand Alone Manager Node. sudo soup

The update appeared completed without any notable errors.  However now after checking the GRID the Manager showed a status "FAULT"

When checking the manager so-status, the elastalert container showed a status "missing".

I tried restarting the elastalert container so-elastalert-restart and it said there was no such container. attached

[so-elastalert-restart.txt](https://github.com/Security-Onion-Solutions/securityonion/files/8594660/so-elastalert-restart.txt)

I tried starting the elastalert container so-elastalert-start and the output didn't show any errors as if it successfully started it?  attached

[so-elastalert-start.txt](https://github.com/Security-Onion-Solutions/securityonion/files/8594661/so-elastalert-start.txt)

Where can I find the logs from soup to see what may have gone wrong?  Am I able to add/remove individual containers to repair them?

**Comment by Mfisher925** [comment, marked as answer]
Hmm, I found a similar discussion previously that noted the answer to move the custom rules out of the rules folder and start the service, this worked... Please close this/delete this discussion. 

Thanks,

**Reply by wally-kroeker** [reply]
Thanks for this.  It lead me to the answer to the problem I was having with the elastalert container not starting.   I had created a new test rule in the /opt/so/rules/elastalert/playbook directory.  I delete the yaml file and ran so-elastalert-start and it came up
